frontend/src/routes/configuration/optionsList.svelte

```diff
@@ -86,8 +98,8 @@
             Name: {name}<br>
             Version: {version}<br>
             Author: {author}
-            <Switch/>
         </div> 
+        <div id="switchContainer"><Switch/></div>
```

Comment:
```suggestion
        <div id="switchWrapper"><Switch/></div>
```

I think it would be more precise